application/comit_node/src/http_api/problem.rs

```diff
@@ -60,17 +63,31 @@ pub fn serialize(e: serde_json::Error) -> HttpApiProblem {
 pub fn not_yet_implemented(feature: &str) -> HttpApiProblem {
     error!("{} not yet implemented", feature);
     HttpApiProblem::with_title_and_type_from_status(StatusCode::INTERNAL_SERVER_ERROR)
+        .set_title("Feature not yet implemented.")
         .set_detail(format!("{} is not yet implemented! Sorry :(", feature))
 }
 
 pub fn action_already_done(action: Action) -> HttpApiProblem {
     error!("{} action has already been done", action);
-    HttpApiProblem::new("action-already-done").set_status(StatusCode::INTERNAL_SERVER_ERROR)
+    HttpApiProblem::with_title_and_type_from_status(StatusCode::INTERNAL_SERVER_ERROR)
+        .set_title("Action already done.")
 }
 
 pub fn invalid_action(action: Action) -> HttpApiProblem {
     error!("{} action is invalid for this swap", action);
-    HttpApiProblem::new("invalid-action").set_status(StatusCode::CONFLICT)
+    HttpApiProblem::with_title_and_type_from_status(StatusCode::CONFLICT)
+        .set_title("Invalid action.")
+        .set_detail("Cannot perform requested action for this swap.")
+}
+
+pub fn unexpected_query_parameters(action: &str) -> HttpApiProblem {
```

Comment:
:x: Instead of passing only the action name, it would be helpful to pass the unexpected query parameters as well.
A custom field in the `HttpApiProblem` seems to be appropriate for this.

https://github.com/chridou/http-api-problem/blob/c8cd8f68af4d5e193011d171104bb00d38db234d/src/lib.rs#L396-L402

Something like:

```rust
set_value("unexpected_parameters", vec![
  ...  
])
```